From 52694698506c23df0897b932ba93108409179302 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 19 Nov 2014 15:28:57 +0800
Subject: [PATCH 389/602] MA-5878 Change GPU device class to gpu_class

change gpu device path from /dev/graphics/galcore to
/dev/galcore due to GPU device class change
graphics_class will be taken as graphics device in android
Only framebuffer device can be taken as graphics device due to
untrusted app cannot have read/write permission on graphics

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 2 +-
 imx6/sepolicy/file_contexts   | 4 +++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index f791224..3659f85 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -17,7 +17,7 @@
 /dev/ttya1                0660   system     gps
 /dev/ptya1                0660   system     gps
 /dev/ion                  0666   media      system
-/dev/graphics/galcore	  0666 	 system     graphics
+/dev/galcore	          0666 	 system     graphics
 /dev/graphics/fb0         0660   system     graphics
 /dev/watchdog             0660   system     root
 /dev/ir                   0660   system     system
diff --git a/imx6/sepolicy/file_contexts b/imx6/sepolicy/file_contexts
index 4ba7e5b..f6423ed 100644
--- a/imx6/sepolicy/file_contexts
+++ b/imx6/sepolicy/file_contexts
@@ -12,4 +12,6 @@
 /dev/ubi0_0                     u:object_r:block_device:s0
 /dev/ubi0_1                     u:object_r:block_device:s0
 /dev/ubi0_2                     u:object_r:block_device:s0
-/dev/ubi0_3                     u:object_r:block_device:s0
\ No newline at end of file
+/dev/ubi0_3                     u:object_r:block_device:s0
+/dev/galcore                    u:object_r:gpu_device:s0
+
-- 
1.8.0

